client.yml

```diff
@@ -2863,6 +2863,130 @@ paths:
       name: id
       in: path
       required: true
+  /services/{id}/edge-routers:
+    get:
+      security:
+      - ztSession: []
+      - oauth2:
+        - openid
+      description: |
+        Retrieves a list of edge router resources that a specific service can dial or bind.
+      tags:
+      - Service
+      summary: List of edge routers that a service can be dialed/bound for the current
+        identity
+      operationId: listServiceEdgeRouters
+      parameters:
+      - type: integer
+        name: limit
+        in: query
+      - type: integer
+        name: offset
+        in: query
+      - type: string
+        name: filter
+        in: query
+      - type: string
+        description: an optional JWT token to validate, if provided a not authorized
```

Comment:
Also reads weird to me...  is this supplemental to the header? does it als0 validate the token? 

how about:
"an optional JWT token use to authenticate the request. If provided, the token needs to be valid else a not authorized response is returned"